src/core/toga/__init__.py

```diff
@@ -12,6 +12,7 @@
 # Widgets
 from .widgets.base import Widget
 
+from .widgets.acitivityindicator import ActivityIndicator
```

Comment:
Typo! `aciticivity...`

Reply:
Oops. At least it was consistent with the file name :)